securedrop/tests/test_encryption.py

```diff
@@ -17,13 +17,14 @@ def test_get_default(self, config):
         assert encryption_mgr
         assert encryption_mgr.get_journalist_public_key()
 
-    def test_generate_source_key_pair(self, setup_journalist_key_and_gpg_folder, source_app):
+    def test_generate_source_key_pair(self, setup_journalist_key_and_gpg_folder,
+                                      source_app, app_storage):
         # Given a source user
         with source_app.app_context():
             source_user = create_source_user(
                 db_session=db.session,
                 source_passphrase=PassphraseGenerator.get_default().generate_passphrase(),
-                source_app_storage=source_app.storage,
+                source_app_storage=app_storage,
```

Comment:
Not super important, but you might now be able to remove the source_app fixture from this test and the one below 🙌

Reply:
Looks like yes to the one below, but no to this one without independently setting up the db and creating an app and passing its context within the test.